SwiftUIHomeworks/SwiftUIHomeworks/ContentView.swift

```diff
@@ -7,20 +7,180 @@
 
 import SwiftUI
 
+/// Wallet screen
 struct ContentView: View {
-    var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+  // MARK: - Private Constants
+  
+  private enum Constants {
+    static let showWalletsText = "Показать все счета"
+    static let currentBalanceText = "Баланс:"
+    static let topUpButtonName = "Пополнить"
+    static let alertTopUpTitle = "Вы уверены?"
+    static let confirmationActionTitle = "Подтвердите действие"
+    static let okButtonText = "Ok"
+    static let fundsValue = 100
+    static let openAccountText = "Открыть счет"
+    static let transferButtonName = "Перевод средств"
+    static let transferAlertTitle = "Подтвердите перевод"
+    static let transferAlertMessage = "Введите код из смс"
+    static let xOffsetCountPlusValue: CGFloat = 200
+    static let xOffsetCountMinusValue: CGFloat = 0
+    static let cornerRadiusValue: CGFloat = 30
+    static let spaceHeightValue: CGFloat = 200
+    static let vStackSpaceValue: CGFloat = 20
+    static let publicWalletName = "Публичный счет:"
+    static let privateWalletName = "Приватный счет:"
+    static let textSize: CGFloat = 20
+    static let textWalletNameSizeValue: CGFloat = 15
+    static let privateBalanceAmount = 654_646
+    static let publicBalanceAmount = 123_424
+  }
+  
+  // MARK: - Public properties
+  
+  var body: some View {
+    VStack(spacing:Constants.vStackSpaceValue) {
+      ZStack {
+        createWalletsText()
+        setupMainView()
+        VStack(spacing: Constants.vStackSpaceValue) {
+          setupBalanceText()
+          setupTopUpButton()
+          createNewWalletButton()
+          createTransferButton()
         }
-        .padding()
+        .offset(x: isShowWallets ? Constants.xOffsetCountPlusValue : Constants.xOffsetCountMinusValue)
+        .animation(.default)
+      }
+      createShowWalletsToggle()
     }
+  }
+  
+  // MARK: - Private properties
+  
+  @State private var isShowWallets = false
```

Comment:
```suggestion
  @State private var isShowWalletsShown = false
```
Если показывает что то то флаг начинаем на is  заканчиваем Shown ниже так же